client/transaction-pool/src/revalidation.rs

```diff
@@ -131,7 +131,7 @@ impl<Api: ChainApi> RevalidationWorker<Api> {
 
 	fn prepare_batch(&mut self) -> Vec<ExHash<Api>> {
 		let mut queued_exts = Vec::new();
-		let mut left = BACKGROUND_REVALIDATION_BATCH_SIZE;
+		let mut left = std::cmp::max(BACKGROUND_REVALIDATION_BATCH_SIZE, self.members.len() / 4);
```

Comment:
This need to be dependant on the revalidation size, otherwise under high load revalidation queue bloats

With this back-exponental it will dissolve faster right after block event comes and slower closer to the next block